engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/repository-details-manage-ostree-branches.controller.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.repositories.controller:RepositoryManageContentController
+ *
+ * @requires $scope
+ * @requries $q
+ * @requires translate
+ * @requires Repository
+ *
+ * @description
+ *   Provides the functionality for the repository details pane.
+ */
+angular.module('Bastion.repositories').controller('RepositoryManageOstreeBranchesController',
+    ['$scope', '$q', 'translate', 'Repository',
+    function ($scope, $q, translate, Repository) {
+
+        $scope.wrap = function (repository) {
```

Comment:
I would expect wrap/unwrap to have similar behaviors - note that one modifies branchObjects on the scope and one returns the data (the latter, with a variable passed in seems better in the long run as you are then creating a function that does one thing and isn't tied to scope)